### ISSUE TITLE
Possible Addition

### ISSUE THREAD

**Issue by flezus** [JustOptimize/return-ShowHiddenChannels#14]
Well, the old SHC had some kind of connection with VoiceActivity, it showed if the person was on a call even if the call was private, could you do that? Or just from the VoiceActivity developer side?

**Comment by JustOptimize** [comment]
Try opening an issue on the VoiceActivity repo, I don't think it's SHC's side but better to be sure.

**Comment by flezus** [comment]
the response I got from the developer was this, is there anything you can do?

### Initially, I was hesitant to allow VoiceActivity to work with ShowHiddenChannels due to my own personal judgement and privacy concerns, but since SHC was an official plugin, I thought it couldn’t do much harm. I asked DevilBro to expose some extra details in SHC so I could access the blacklist, settings, etc. in VoiceActivity.

Now that SHC is not an “official” plugin on the BetterDiscord website, I decided not to keep supporting the plugin’s ability to “work together” with SHC. Even if SHC was allowed in BD’s guidelines, DevilBro wanted to discontinue it, and I will respect that decision. I myself am not going to put effort into adding support for another plugin that really shouldn’t exist anymore.

That being said, there is nothing stopping you or another developer from forking an unofficial version of VoiceActivity that has support for the new SHC (while following my MIT license, of course). It wasn’t at all a difficult change to make, and I’m sure it could easily be brought back or replicated. Obviously, at that point the forker would be responsible for updating and maintaining the plugin.

TL;DR: No, I will not be bringing it back. No, I will not be providing support for the plugins working together. Yes, you can fork it and maintain your own version with the new ShowHiddenChannels support if you so wish.

**Comment by JustOptimize** [comment]
I can't do anything at the moment, but maybe in the future if someone else intends to create a fork of that plugin.
